### PR TITLE
add example tests to travis

### PR DESCRIPTION
**Pull request by lmcgupe**
Signed-off-by: guy perron <guy.perron@ericsson.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "theia-core",
-  "version": "0.0.1-alpha.2",
+  "version": "0.0.1-alpha.1",
```
Comment (akosyakov) [inline package.json]: it looks bogus

**Comment by hexa00** [comment]
I think you did a pull request of the wrong branch,

Also the elelctron-mocha command should be an npm script

File: package.json (hunk 1 above)
Comment (lmcgupe) [inline package.json]: It's been removed, it was updated automatically after running npm install electron-mocha --save

**Comment by lmcgupe** [comment]
comments were adressed, I still get an error when travis runs the build on not being able to find bunyan library. It's referenced int the package.json, is there another place when to tell travis about it ? it runs fine locally.

**Comment by kittaakos** [comment]
Let me check.

**Comment by kittaakos** [comment]
I think the build failure is unrelated to the `bunyan` library. Or I am simply just blind. Please try to mimic Travis-CI locally by running: `./config/scripts/clean-all.sh && ./config/scripts/travis-ci-build.sh`.

I think the lack of type information for Electron causes your problem. Let me check out your branch and run it locally.

**Comment by kittaakos** [comment]
@lmcgupe Yeah, I can reproduce the build issue locally. Do not you mind if I make some modifications directly on your `test` branch? I wait for your approval.

**Comment by lmcgupe** [comment]
go ahead

**Comment by kittaakos** [comment]
Roger that!

**Comment by lmcgupe** [comment]
i was just able to run it successfully locally ...

**Comment by kittaakos** [comment]
With the commands I provided above? Strange. I switched to your branch, executed the command, and it failed with the same error I can see in the Travis-CI log.

**Comment by lmcgupe** [comment]
yes, it ends with my test passing ...

**Comment by lmcgupe** [comment]
can you see node_modules/bunyan ?

**Comment by kittaakos** [comment]
For the example or for Theia itself? I cannot find it in Theia, but it is not in the `package.json` either. It is declared only for the examples as I see.

**Comment by hexa00** [comment]
It is in theia also, but your pull request is wrong ... we'll discuss this offline a bit

**Comment by kittaakos** [comment]
I still have problems with `electron`. Let me check the versions.

**Comment by kittaakos** [comment]
> your pull request is wrong ... we'll discuss this offline a bit

What does that mean? Should I just simply stop investigating this issue?

**Comment by lmcgupe** [comment]
antoine and i are looking into it. don't spend time on it for now. thanks

________________________________

**Pull request closed without merging**